### PR TITLE
JENKINS-66008 Fixing SCA Agent-Based scan issues

### PR DESCRIPTION
**Pull request by sbuhary**
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Review by dennisgu1 — APPROVED** [review]
The only one left is log4j SCA vulnerability, and we will need to mitigate it, am I right?
Also please run our Jenkins automation and make sure it pass, thanks.

**Comment by sbuhary** [comment]
> The only one left is log4j SCA vulnerability, and we will need to mitigate it, am I right?
> Also please run our Jenkins automation and make sure it pass, thanks.

@dennisgu1 Yes, we need to mitigate log4j SCA vulnerability. Also Jenkins plugin automation is already executed and it has passed.

File: pom.xml [hunk 1]
```diff
@@ -94,6 +98,72 @@
...
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-webmvc</artifactId>
+            <version>5.3.8</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>30.0-android</version>
```
Comment (kkyiu) [inline pom.xml:111]: According to this page: https://guava.dev/, the "-android" is for Android apps. Please consider using "30.0-jre" for our plugin.

Comment (sbuhary) [reply pom.xml]: This is changed and tested. Thank you.

**Review by dennisgu1 — APPROVED** [review]

**Review by kkyiu — APPROVED** [review]

**Pull request merged**